cookbook/extending-website-search.rst

```diff
@@ -0,0 +1,35 @@
+Extending website search
+=========================
+
+There is the possibility to find other documents than pages
+through the website search, but that needs to be configured.
+The SuluArticleBundle is probably one of the best use cases
+for that, because you might want articles to appear in the
+website search.
+
+The configuration looks like the following:
+
+.. code-block:: yaml
+
+    # config/packages/sulu_search.yaml
+
+    sulu_search:
+        website:
+            indexes:
+                articles: article_published
```

Comment:
Shouldn't the articles be preprended by the `SuluArticleBundle`? Why should we burden the developer with doing this on their own?

Reply:
Personally I would consider this a bug fix, not a BC break, therefore I would not care about the changed behavior.